### DISCUSSION TITLE
I read the docs but don’t understand how to access images via middleware

### DISCUSSION THREAD

**Discussion by kgrosvenor** [SixLabors/ImageSharp.Web#325] [Q&A]
How does AWS provider work url structure wise, how does one access the bucket once you’ve configured in start up?

I feel documentation could have more examples.

kind regards

**Comment by JimBobSquarePants** [comment]
It’s the bucket name followed by the image name. What’s missing?

https://docs.sixlabors.com/articles/imagesharp.web/imageproviders.html#awss3storageimageprovider

**Comment by kgrosvenor** [comment]
Thanks @JimBobSquarePants ,  i didn't see the dropdown menu on the website from mobile, so guess its a skill issue on my end, now i'm on the PC i see them 😄  

Apologies
